lib/handlers/ttt.ex

```diff
@@ -0,0 +1,1042 @@
+defmodule HTTPServer.Handlers.TTT do
```

Comment:
What is this file? 🤔

Reply:
Oops! I thought that got deleted! It's HTML for a tic tac to game that Ian and I were testing yesterday 😅

Reply:
haha I was thinking it had to be something like that 🤣